types/fraudproof.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmcrypto "github.com/tendermint/tendermint/proto/tendermint/crypto"
+)
+
+// Represents a single-round fraudProof
+type FraudProof struct {
+	// The block height to load state of
+	blockHeight int64
+
+	appHash []byte
```

Comment:
I think appHash should be documented as well.

Reply:
Currently removed this updated structure of Fraud Proof in order to avoid adding serialization issues in this PR and making it bigger. Tracking issue [here](https://github.com/celestiaorg/rollmint/issues/551).